plugins/out_retry/retry.c

```diff
@@ -80,12 +72,12 @@ static void cb_retry_flush(const void *data, size_t bytes,
     ctx = out_context;
     ctx->count++;
 
-    if (ctx->count <= ctx->n_retry) {
-        flb_plg_debug(ctx->ins, "retry %i/%i", ctx->count, ctx->n_retry);
+    if (ctx->count <= ctx->retries) {
+        flb_debug("[retry] retry %i/%i", ctx->count, ctx->retries);
         FLB_OUTPUT_RETURN(FLB_RETRY);
     }
     else {
-        flb_plg_debug(ctx->ins, "flush", ctx->count, ctx->n_retry);
```

Comment:
note that this change is reverting our improvement in the logging api (same thing above)

Reply:
@edsiper okay. So what changes shall I make